### PR TITLE
Adds Rainbow Alert

### PR DESCRIPTION
**Pull request by Kurfursten**
untitled

Alright, we've had our fun. Too many rounds have been gunked by clowns with all access requesting more clowns. It's an immediate gunk effect. Last time it ended with nonantag clowns stealing mechs and polyaciding people.

You can still do it, but requesting additional mimes or clowns beyond the base level requires a rainbow alert.

🆑 
*rscadd: Requesting additional clowns and mimes now requires a rainbow alert.

### CODE REVIEW

**Comment by ghost** [comment]
Can you add requesting additional captains/heads of security behind red alert?

Wouldn't a clown with all access be able to just set rainbow alert?

**Comment by Kurfursten** [comment]
Sure, get some 👍 s on your comment and I'll add it

**Comment by ghost** [comment]
The juxtaposition in the image with >Assistant (0/99) did give me a good laugh, though.

But anyway, not really a terrible idea, I think, but maybe make it so you can recruit one or two more _(one sounds preferable)_ without said alert level. Two clowns or two mimes aren't likely to irreperably gunk the round, and being able to replace a lost entertainer _(>implying anyone would actually use it for that)_ would be nice.

@codeungabunga It's tied to the keycard device, so he'd have to have some willing lackey _(or incredible speed with the hand tele)_ to do it. Considering everyone loves clowns _(they don't)_ it's likely they could, though — I think it'll at least stem it to some degree.

File: code/modules/security levels/security levels.dm [hunk 1]
```diff
@@ -18,8 +20,16 @@
...
+				//Attention! Code rainbow!
+				//There is an immediate serious lack of threats to the station. The clown may have bananas unholstered at all times. The Head of Personnel may now request additional entertainment staff.
+				var/dat = {"<font size=4><html><div><span style="color:#ff0000;">A</span><span style="color:#ff1900;">t</span><span style="color:#ff3300;">t</span><span style="color:#ff4c00;">e</span><span style="color:#ff6600;">n</span><span style="color:#ff7f00;">t</span><span style="color:#ff9f00;">i</span><span style="color:#ffbf00;">o</span><span style="color:#ffdf00;">n</span><span style="color:#ffff00;">!</span><span style="color:#ccff00;"> </span><span style="color:#99ff00;">C</span><span style="color:#66ff00;">o</span><span style="color:#33ff00;">d</span><span style="color:#00ff00;">e</span><span style="color:#00ff40;"> </span><span style="color:#00ff80;">r</span><span style="color:#00ffbf;">a</span><span style="color:#00ffff;">i</span><span style="color:#00ccff;">n</span><span style="color:#0099ff;">b</span><span style="color:#0066ff;">o</span><span style="color:#0033ff;">w</span><span style="color:#0000ff;">!</span></div></font>
```
Comment (boy2mantwicethefam) [inline code/modules/security levels/security levels.dm:29]: OH LAWD wouldn't it be easier to turn this into its own thing? span style="rainbow" or something

Comment (ghost) [reply code/modules/security levels/security levels.dm:29]: hahaha oh jesus i didn't even see that
I agree.

Comment (Kurfursten) [reply code/modules/security levels/security levels.dm:29]: Quite literally wyci

I don't think it's doable to do what you're asking, or if it is it requires more advanced knowledge of html than I have.

Comment (mtpowell) [reply code/modules/security levels/security levels.dm:29]: It'd be easy with CSS and using :nth-letter(#), even easier with Javashit

Comment (Kurfursten) [reply code/modules/security levels/security levels.dm:29]: Great, I'll rebase after someone else writes the css

**Comment by Kurfursten** [comment]
ATTENTION

#21853
#21854

**Pull request merged**